app.json

```diff
@@ -29,6 +30,11 @@
     },
     "ON_HEROKU": {
       "value": "true"
+    },
+    "ENCRYPTION_KEY": {
+      "value": "ABCGDHEUSHFKIRMJSDFIUTEHFNGOWPOR",
```

Comment:
You may test this by clicking the `Deploy to Heroku` button in https://github.com/imujjwal96/yaydoc/tree/issue180 's Readme file